enos/enos-scenario-e2e-aws-rdp-target.hcl

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+scenario "e2e_aws_rdp_target" {
+  terraform_cli = terraform_cli.default
+  terraform     = terraform.default
+  providers = [
+    provider.aws.default,
+    provider.enos.default
+  ]
+
+  matrix {
+    rdp_server = ["2016", "2019", "2022", "2025"]
+    ip_version = ["4", "dual"]
+  }
+
+  locals {
+    build_path = {
+      "local" = "/tmp",
+      "crt"   = var.crt_bundle_path == null ? null : abspath(var.crt_bundle_path)
+    }
+
+    tags = merge({
+      "Project Name" : var.project_name
+      "Project" : "Enos",
+      "Environment" : "ci"
+    }, var.tags)
+
+    collocated_tag = "collocated"
```

Comment:
There's still a couple of things that aren't used here
- `build_path`
- `collocated_tag`